src/dbos-executor.ts

```diff
@@ -107,7 +107,13 @@ export class DBOSExecutor {
   /* WORKFLOW EXECUTOR LIFE CYCLE MANAGEMENT */
   constructor(readonly config: DBOSConfig, systemDatabase?: SystemDatabase) {
     this.debugMode = config.debugProxy ? true : false;
-    this.logger = createGlobalLogger(this.config.telemetry?.logs);
+    const telemetryExporters: TelemetryExporter[] = [];
+    for (const exporterConfig of config.telemetry?.OTLPExporters ?? []) {
```

Comment:
Logically, we only have a single exporter which has some number of endpoints (some for traces, some for logs).